CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## TBD
+### Fixed
+- `apiary-metastore-listener` now serialises metastore table properties as a Map instead of a String. 
```

Comment:
serialize vs serialise ?

Reply:
Yep you're right we use the US spelling, thanks!